qa/build/scala.Dockerfile

```
@@ -79,10 +89,8 @@
 # Install trcli
 RUN pip3 install --break-system-packages trcli
 
-# Install global Node/Yarn dependencies
 # WORKDIR /home/qa/code
-RUN yarn install && \
-    yarn global add junit-report-merger
+RUN yarn install
```

Hadolint (2.12.0)

[info] `yarn cache clean` missing after `yarn install` was run.

(DL3060)